Tests/configure_and_test_integration_instances.py

```diff
@@ -1066,76 +1038,65 @@ def get_pack_ids_to_install():
 
 
-def install_nightly_pack(build, prints_manager):
-    threads_print_manager = ParallelPrintsManager(len(build.servers))
-    nightly_install_packs(build, threads_print_manager, install_method=install_all_content_packs)
+def install_nightly_pack(build):
+    nightly_install_packs(build, install_method=install_all_content_packs)
     create_nightly_test_pack()
-    nightly_install_packs(build, threads_print_manager, install_method=upload_zipped_packs,
+    nightly_install_packs(build, install_method=upload_zipped_packs,
                           pack_path=f'{Build.test_pack_target}/test_pack.zip')
 
-    prints_manager.add_print_job('Sleeping for 45 seconds...', print_warning, 0, include_timestamp=True)
-    prints_manager.execute_thread_prints(0)
+    logging.info('Sleeping for 45 seconds while installing nightly packs')
     sleep(45)
 
 
-def install_packs(build, prints_manager, pack_ids=None):
+def install_packs(build, pack_ids=None):
     pack_ids = get_pack_ids_to_install() if pack_ids is None else pack_ids
     installed_content_packs_successfully = True
     for server in build.servers:
         try:
-            _, flag = search_and_install_packs_and_their_dependencies(pack_ids, server.client,
-                                                                      prints_manager)
+            _, flag = search_and_install_packs_and_their_dependencies(pack_ids, server.client)
             if not flag:
                 raise Exception('Failed to search and install packs.')
-        except Exception as exc:
-            prints_manager.add_print_job(str(exc), print_error, 0)
-            prints_manager.execute_thread_prints(0)
+        except Exception:
+            logging.exception('Failed to search and install packs')
             installed_content_packs_successfully = False
 
     return installed_content_packs_successfully
 
 
-def configure_server_instances(build: Build, tests_for_iteration, all_new_integrations, modified_integrations,
-                               prints_manager):
+def configure_server_instances(build: Build, tests_for_iteration, all_new_integrations, modified_integrations):
     all_module_instances = []
     brand_new_integrations = []
     testing_client = build.servers[0].client
     for test in tests_for_iteration:
         integrations = get_integrations_for_test(test, build.skipped_integrations_conf)
 
-        integrations_names = [i.get('name') for i in integrations]
-        prints_manager.add_print_job('All Integrations for test "{}":'.format(test.get('playbookID')), print_warning, 0)
-        prints_manager.add_print_job(integrations_names, print_warning, 0)
+        playbook_id = test.get('playbookID')
 
         new_integrations, modified_integrations, unchanged_integrations, integration_to_status = group_integrations(
             integrations, build.skipped_integrations_conf, all_new_integrations, modified_integrations
         )
-
+        integration_to_status_string = '\n\t\t\t\t\t\t'.join(
```

Comment:
why did you add all the tabs only here?
you don't do it in any other multiline log

Reply:
All the other multiline logs happens only once.
these logs haapen as the number of packs that should be installed and it makes the logs a bit confusing.
Look at the example logs and tell me what you think..